circuit-foundation/src/commonMain/kotlin/com/slack/circuit/foundation/NavigableCircuitContent.kt

```diff
@@ -262,33 +268,33 @@ public object NavigatorDefaults {
       return enterTransition togetherWith exitTransition
     }
 
-    private fun AnimatedContentTransitionScope<*>.transitionFor(diff: Int): ContentTransform {
```

Comment:
This function felt a bit unnecessary? Happy to add it back in though.

Reply:
Yeah it was a bit of a leftover from when I was comparing the new to the old 👌